frontend/src/console/components/Hero/Hero.tsx

```diff
@@ -82,62 +82,78 @@ export class HeroComponent extends React.PureComponent<Props, State> {
 
     this.state = {
       title: '',
-      subtitle: ''
+      subtitle: '',
+      renderBool: true
     };
   }
 
   public componentDidMount() {
     this.checkAndUpdateState();
   }
 
-  public componentDidUpdate() {
+  public componentDidUpdate(prevProps: Props) {
     this.checkAndUpdateState();
+    if (
+      this.staticContentOverridesExists() &&
+      this.staticContentOverridesExists(prevProps) &&
+      prevProps.activeInstance.data &&
+      this.props.activeInstance.data &&
```

Comment:
I tried doing it without the checks, and it doesn't compiles, so that's why I left them there :)

Reply:
That's because TypeScript doesn't resolve this nested check (despite it actually exists) and that's why we should use the `!` here, as indicated in [this comment](https://github.com/open-craft/opencraft/pull/572#discussion_r421738649). Related part of that snippet:
```ts
if (
    this.staticContentOverridesExists() &&
    this.staticContentOverridesExists(prevProps) &&
    prevProps.activeInstance.data!.draftStaticContentOverrides !==
      this.props.activeInstance.data!.draftStaticContentOverrides
  ) {
    this.checkNewAndUpdateState();
  }
```

Reply:
@Agrendalath  thanks for the comment, and the snippet!
I fixed it :smile: 
Now it works perfect :+1: